javascript/script.js

```diff
@@ -8,20 +10,20 @@ let logConsoleMessage = () => {
     "font-size: 14px; color: #3399FF; text-decoration: underline;"
   );
 };
-let createWebringList = (sites) => {
+let createWebringList = (matchedSiteIndices) => {
```

Comment:
im passing in list of matched indicies instead the actual objects so it makes the comparison on line 24 easier (otherwise we'd have to re-compare the objects w/fuzzy matching to get a determinate result)